lib/turn.rb

```diff
@@ -0,0 +1,22 @@
+class Turn
+  attr_reader :guess,
+              :card
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    @guess == @card.answer
+  end
+
+  def feedback
+    if @guess != @card.answer
+      "Incorrect."
+    else
+      "Correct!"
+    end
+  end
```

Comment:
On line 15, this code is identical to the code you have in the `correct?` method. Because of that, you can replace line 15 to just read `if correct?`.